### PR TITLE
Remove focus border for data inspector and add ide_launched dimension.

### PR DESCRIPTION
**Pull request by terrylucas**
- Don't display border selection for the data inspector used by both debugger and memory views.
- Added `ide_launched` dimension for GA.
- Fixed dark mode theme for find memory popup, allocation/field hover card and memory navigation history links.
- Removed echoing of devToolsReady to console.

### CODE REVIEW

File: packages/devtools/lib/src/ui/analytics.dart [hunk 1]
```diff
@@ -137,14 +145,16 @@ void screen(
...
+        event_category: screenViewEvent,
+        value: value,
+        user_app: userAppType,
+        user_build: userBuildType,
+        user_platform: userPlatformType,
+        devtools_platform: devtoolsPlatformType,
+        devtools_chrome: devtoolsChrome,
+        devtools_version: devtoolsVersion,
+        ide_launched: ideLaunched,
+      ));
```
Comment (jacob314) [inline packages/devtools/lib/src/ui/analytics.dart]: nit: for flutter style add a trailing comma between `))` as well.

File: packages/devtools/lib/src/ui/analytics_platform.dart [hunk 2]
```diff
@@ -61,6 +61,16 @@ void computeDevToolsCustomGTagsData() {
   }
 }
 
+// Look at the query parameters '&ide=' and record in GA.
+void computeDevToolsQueryParams() {
+  ga.ideLaunched = ga.ideLaunchedCLI; // Default is Command Line launch.
+  final searchParams = html.UrlSearchParams(html.window.location.search);
```
Comment (jacob314) [inline packages/devtools/lib/src/ui/analytics_platform.dart]: use
`dart
    final Uri uri = Uri.parse(window.location.toString());
    uri.queryParameters[ga.ideLaunchedQuery];
  ` 
to make the code a little closer to working with flutter:web

File: packages/devtools/lib/src/ui/analytics.dart (hunk 1 above)
Comment (terrylucas) [reply packages/devtools/lib/src/ui/analytics.dart]: Go it - done.

File: packages/devtools/lib/src/ui/analytics_platform.dart (hunk 2 above)
Comment (terrylucas) [reply packages/devtools/lib/src/ui/analytics_platform.dart]: Done.

File: packages/devtools/lib/src/model/model.dart [hunk 3]
```diff
@@ -79,9 +79,7 @@ class App {
     js.context['devtools'] = binding;
   }
 
-  Future<void> devToolsReady(dynamic message) async {
-    _sendNotification('app.devToolsReady', message);
```
Comment (jacob314) [inline packages/devtools/lib/src/model/model.dart]: hmm. I'm not sure this is the right place to delete it.
I think test code is depending on the 'app.devToolsReady' event.
Can we just tweak things so it doesn't get displayed in the console?

Comment (terrylucas) [reply packages/devtools/lib/src/model/model.dart]: Yes, just found that out - travis failed.  I'm not sure _sendNotification should ever print to the console?  Think this print should be commented out for now.  With a TODO number of other things also print to console in model.dart.

**Pull request merged**